packages/babel-preset/index.js

```diff
@@ -1 +1 @@
-module.exports = require('./web');
+module.exports = require('./common');
```

Comment:
As we're making a breaking change anyway could we move  the contents of common.js into index.js? and then we can reference the preset `@shopify/babel-preset` rather than `@shopify/babel-preset/common`